qiskit_ibm_runtime/estimator.py

```diff
@@ -189,6 +193,17 @@ def __init__(
             backend = session or backend
             self._session = get_default_session(service, backend)
 
+        self._first_run = True
+        self._circuits_map = {}
+        if self.circuits:
+            for circuit in self.circuits:
+                circuit_id = _hash(
```

Comment:
Why do you need to hash it when the output of `json.dumps()` is already a string?

Reply:
we used `_hash` because the value returned from `json.dumps()` was too long. We can remove it if you want.